compiler/compiler.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/tinygo-org/tinygo/ir"
 	"github.com/tinygo-org/tinygo/loader"
 	"golang.org/x/tools/go/ssa"
-	"tinygo.org/x/go-llvm"
+	llvm "tinygo.org/x/go-llvm"
```

Comment:
This change (and a few similar ones below) are not related to rpi3 support, and should be removed.